eclipse-collections-testutils/src/test/java/org/eclipse/collections/impl/test/ExceptionThrownTest.java

```diff
@@ -44,12 +42,12 @@ public void callableException()
     {
         try
         {
-            Verify.assertThrows(NullPointerException.class, new EmptyCallable());
+            assertThrows(NullPointerException.class, EmptyCallable::new);
```

Comment:
This looks like it was intended to test `Verify.assertThrows()` and even though it doesn't make much sense anymore, because the method was removed, keeping the test the way it was asserts source backwards compatibility. Is it a linter violation to keep it? If it works, I think we should keep `Verify` here.

Reply:
done - cast is needed though because the type between Verify.assertThrows and the Assertions.throws doesn't match and the method call becomes ambiguous.